sdk/keyvault/azure-security-keyvault-certificates/src/certificate_client_operations.cpp

```diff
@@ -30,7 +30,7 @@ Azure::Response<CertificateOperationProperties> CreateCertificateOperation::Poll
 std::unique_ptr<Azure::Core::Http::RawResponse> CreateCertificateOperation::PollInternal(
     Azure::Core::Context const& context)
 {
-  std::unique_ptr<Azure::Core::Http::RawResponse> rawResponse;
+  std::unique_ptr<Azure::Core::Http::RawResponse> rawResponse = std::move(m_rawResponse);
```

Comment:
This assignment doesn't seem necessary. If we are doing it just for the sake of consistency with other implementations that do actually need it, consider adding a comment as to why we are keeping it.

Reply:
Or, I guess, the more relevant question is, why don't we have a `if (!IsDone())` block here, when we do have it in other `PollInternal` implementations?